src/pow.cpp

```diff
@@ -147,7 +147,8 @@ unsigned int GetNextWorkRequired(const CBlockIndex* pindexLast, const CBlockHead
         else retarget = DIFF_BTC;
     // testnet -- we want a lot of coins in existance early on 
     } else {
-        if (pindexLast->nHeight + 1 >= 3000) retarget = DIFF_DGW;
+        if (pindexLast->nHeight + 1 >= 80000) retarget = DIFF_KGW;
```

Comment:
It looks like this switches to KGW after 80K.  Is that what we want ?  It doesn't seem to match the commit message.